active_learning_multiannotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,461 @@
+    "This notebook implements the following steps:\n",
+    "\n",
+    "1. Establish some already labeled data. Use this data to train a classifier model and then obtain out-of-sample predicted probabilities for each labeled and unlabeled example.\n",
+    "2. Compute active learning scores for every example, which estimate our current confidence in knowing its true label.\n",
+    "3. Collect additional labels for the unlabeled examples with the lowest active learning scores. These are the most potentially informative examples whose true label we are least certain of. \n",
+    "4. Repeat the steps above to collect as many labels as your budget permits.\n",
+    "\n",
+    "The accuracy of the model trained on the resulting dataset will generally match that of the same model trained on a much larger set of randomly collected labels. I.e. this is the most cost-effective way to train an accurate classifier!\n",
+    "\n",
+    "In this example we use the caltch256 dataset combined with autogluon's new MultiModalPredictor architecture."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import dependencies and get data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/ubuntu/.local/lib/python3.8/site-packages/tqdm/auto.py:22: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+      "  from .autonotebook import tqdm as notebook_tqdm\n"
+     ]
+    }
+   ],
+   "source": [
+    "import time\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from autogluon.multimodal import MultiModalPredictor\n",
+    "from autogluon.vision import ImageDataset\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from cleanlab.multiannotator import get_majority_vote_label, get_label_quality_multiannotator, get_active_learning_scores\n",
+    "from utils.model_training_autogluon import cross_val_predict_autogluon_classification_dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We load the following datafiles:\n",
+    "\n",
+    "- `dataset` is a DataFrame that contains labels and image paths for each example\n",
+    "\n",
+    "We will then randomly split the dataset into train and test splits. Test data will just be used to measure the accuracy in our model after each active learning round. The train data will further be split into a labeled and unlabeled part. For the df_labeled we will use the labels to train the model while the df_unlabeled will simulate active learning by allowing us to artifically insert mode labeled data in between rounds.\n",
```

Comment:
```suggestion
    "We then randomly split the dataset into train and test splits. Test data are just used to measure the accuracy in our model after each active learning round (you may not have this in your applications).  The train data will further be split into labeled and unlabeled pools. For the df_labeled we will use the labels to train the model while the df_unlabeled will simulate active learning by allowing us to artificially insert more labeled data in between rounds.\n",
```

Reply:
I didn't fix for you above, but the variable names like df_labeled should be written as: \`df_labeled\`